### PR TITLE
NamespaceMap: Remove TODO and fix caps

### PR DESCRIPTION
**Pull request by bact**
- Remove "TODO" as the URLs are already updated. (although they are all pointing directly to the repo: https://github.com/spdx/spdx-3-model/ - is this something we are preferred?)
- Also use 4-space indent for list, to keep the multiple paragraphs rendered correctly.

### CODE REVIEW

**Comment by goneall** [comment]
@bact I personally think we should point to the generated mkdocs HTML pages rather than the repo.  We can discuss on the next tech call.

**Review by kestewart — APPROVED** [review]
Thanks!

**Comment by bact** [comment]
> @bact I personally think we should point to the generated mkdocs HTML pages rather than the repo. We can discuss on the next tech call.

@goneall Our current "Model and serializations" page at https://spdx.github.io/spdx-spec/v3.0/serializations/ does not has information about NamespaceMap. While our "Notes on Serialization" at https://github.com/spdx/spdx-3-model/tree/main/serialization#readme has it.

**Comment by goneall** [comment]
Discussed on 30 July 2024 tech call:
- Agreed we should point to the HTML - required for ISO
- We need to open a pull request to add the information to the spec serialization section, then change the links in this PR

**Comment by bact** [comment]
>     * Agreed we should point to the HTML - required for ISO
> 
>     * We need to open a pull request to add the information to the spec serialization section, then change the links in this PR

A PR #1016 is open.

**Review by kestewart — APPROVED** [review]

File: model/Core/Classes/NamespaceMap.md [hunk 1]
```diff
@@ -15,38 +15,39 @@ order to provide a more human-readable and smaller serialized representation of
 the Elements.
 
 For details of how NamespaceMap content is to be serialized please refer to
-general SPDX serialization guidance at
-<https://github.com/spdx/spdx-3-model/blob/main/serialization/README.md>
-and the various serialization format specific .md filed under
-<https://github.com/spdx/spdx-3-model/tree/main/serialization>
-(TODO: update the URLs as soon as the context is publicly available)
+[Model and Serializations](../../../docs/serializations.md)
```
Comment (zvr) [inline model/Core/Classes/NamespaceMap.md:18]:
```suggestion
the [Model and Serializations](../../../docs/serializations.md) clause
```

File: model/Core/Classes/NamespaceMap.md [hunk 2]
```diff
@@ -15,38 +15,39 @@ order to provide a more human-readable and smaller serialized representation of
...
+and the various serialization format-specific .md files in
```
Comment (zvr) [inline model/Core/Classes/NamespaceMap.md:19]:
```suggestion
and the various serialization format-specific files in
```

**Review by zvr — CHANGES_REQUESTED** [review]

**Comment by bact** [comment]
@zvr @kestewart this PR is automatically closed by GitHub because the description of PR #1016 has the word "fix" before this PR number. Do you mind to reopen this please? Thank you. This is probably one of the few last PRs for 3.0.1 model.

**Comment by bact** [comment]
Reopened as #821, with a new fix

**Pull request closed without merging**